### PR TITLE
conda-smithy 3.30.0

### PR DESCRIPTION
**Pull request by isuruf**
Merge only after success.

This pull request was auto-generated by [rever](https://regro.github.io/rever-docs/)

### CODE REVIEW

**Review by jakirkham — APPROVED** [review]
Thanks Isuru! 🙏

File: .scripts/build_steps.sh [hunk 1]
```diff
@@ -34,9 +34,9 @@ CONDARC
 export CONDA_LIBMAMBA_SOLVER_NO_CHANNELS_FROM_INSTALLED=1
 
 mamba install --update-specs --yes --quiet --channel conda-forge --strict-channel-priority \
-    pip mamba conda-build conda-forge-ci-setup=4
+    pip mamba conda-build boa conda-forge-ci-setup=4
 mamba update --update-specs --yes --quiet --channel conda-forge --strict-channel-priority \
-    pip mamba conda-build conda-forge-ci-setup=4
+    pip mamba conda-build boa conda-forge-ci-setup=4
```
Comment (h-vetinari) [inline .scripts/build_steps.sh:39]: Let's hope that the interaction of `boa` with `conda-forge-ci-setup=4` goes smoothly (first time we're trying this AFAIK) 🤞

**Review by h-vetinari — APPROVED** [review]

File: .scripts/build_steps.sh (hunk 1 above)
Comment (isuruf) [reply .scripts/build_steps.sh:39]: It's been tested at https://github.com/conda-forge/cf-autotick-bot-test-package-feedstock/pull/489

Comment (h-vetinari) [reply .scripts/build_steps.sh:39]: I mean broadly across many feedstocks. There were a bunch of incompatibilities between mamba, boa & conda-build, let's hope they all got fixed in the recent round of releases and that no others come up.

**Pull request merged**